test/nyc-integration.js

```diff
@@ -693,6 +693,23 @@ describe('the nyc cli', function () {
         })
       })
 
+      it('can instrument the root directory', function (done) {
```

Comment:
I know it's just a test but can we say call it the project directory instead of the root directory?